### PR TITLE
docs(navigation): adds height constraint to css demos

### PR DESCRIPTION
**Pull request by hippee-lee**
Updated navigation code snippits to use doc-demo
Adds a constraint class that can be combined with main-container to limit height to content size

Signed-off-by: Matt Hippely <mhippely@vmware.com>

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)
- [x] If applicable, have a visual design approval

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [x] clarity.design website / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?
Change to the home page for new website. 
Simplified transition between hero and content. 

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by bbogdanov** [comment]
lgtm

File: apps/website/foundation/themes/README.md [hunk 1]
```diff
@@ -226,7 +207,7 @@ Next, we need to include the the application scss variable overrides and the Cla
 @import "../node_modules/@clr/ui/src/utils/components.clarity";
 ```
 
-##### Why is SASS-based theming going away?
+##### Why are SASS-based theming going away?
```
Comment (mathisscott) [inline apps/website/foundation/themes/README.md:210]: Unfortunately `Why is...` is grammatically correct. Can you revert this change?

Comment (hippee-lee) [reply apps/website/foundation/themes/README.md]: Reverted. Good catch.

File: apps/website/foundation/themes/README.md [hunk 2]
```diff
@@ -193,7 +174,7 @@ SASS-based theming is deprecated in Clarity 3.0 and will no longer work when Cla
 
 SASS-based theming is most familiar to those who use SASS/SCSS in their own products to build CSS stylesheets. SASS acts like a superset of CSS that enables convenient nesting syntax and programming-like functionality in a CSS-based language that compiles to CSS. SASS also gives developers variables they can reuse throughout their SASS codebase.
 
-Clarity has a number of such variables. The [\_variables.clarity.scss](https://github.com/vmware/clarity/blob/master/src/clr-angular/utils/_variables.clarity.scss) file in the Clarity project can serve as a directory for where all of the variable files can be found.
+Clarity has a number of such variables. The [\_variables.clarity.scss](https://github.com/vmware/clarity/blob/master/packages/angular/projects/clr-angular/src/utils/_variables.clarity.scss) file in the Clarity project can serve as a directory for where all of the variable files can be found.
```
Comment (gnomeontherun) [inline apps/website/foundation/themes/README.md:177]: Backslash in `[\_variables.clarity.scss]`?

Comment (hippee-lee) [reply apps/website/foundation/themes/README.md:177]: The backslash is added by prettier. [playground example](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEBtA+gMwJYBs4B0AzmEUQLoAUu2UA1gJQgA0IEADjNtEcqAIYAnQRADuABSEJeKfgDcI2ACYsQAI0H8wdODADK7LbQDmyGIICucVgAsYAW1wB1G9nhFDYOHulvsctwBPZHAyVVoiOEEYcU1je35kTH5cSNYAKyIADwAhTW1dPX57OAAZWjgklLSQTKy9E3wARQsIeCrU6xBDQUjBEITBOiUxKFV2QVoYJ2UYG2QADgAGVgmISKdNdhCJuD65StYAR1b4WI4ZEH4iAFooODglR9VBOBPsV9j+eMSkZM7WJF7NgzJYukRGnAWm1Kn9ql0YPw1DMlHNkAAmVjmfh4EwAYQg9gSISg0EOIAskQAKkiZP8anIrABJKBPWB6MCTTgAQVZehggXwHUiAF8RUA) It doesn’t break the link that gets generated when markdwon is rendered out.

I can live with it. Prettier doesn't let us ignore specific lines, only files so the only other option we have is to start ignoring markdown files.
